### PR TITLE
[slack] Update files metric

### LINKED ISSUE #867: [slack] FIles are not getting enriched
The enricher for makes an attempt to enrich the files which are sent as messages:
https://github.com/chaoss/grimoirelab-elk/blob/ac8df68d889a7a4eb2b5f0c693364eeafcb3ec80/grimoire_elk/enriched/slack.py#L146

However, the response returned from Perceval is the field `files` which is a list of file JSONs. FInd sample response below
```
{
    "backend_name": "Slack",
    "backend_version": "0.9.0",
    "category": "message",
    "classified_fields_filtered": null,
    "data": {
        "blocks": [
            {
                "block_id": "cLcLv",
                "elements": [
                    {
                        "elements": [
                            {
                                "text": "Uploading test file",
                                "type": "text"
                            }
                        ],
                        "type": "rich_text_section"
                    }
                ],
                "type": "rich_text"
            }
        ],
        "channel_info": {
            "created": 1588076799,
            "creator": "<redacted>",
            "id": "<redacted>",
            "is_archived": false,
            "is_channel": true,
            "is_ext_shared": false,
            "is_general": false,
            "is_group": false,
            "is_im": false,
            "is_member": true,
            "is_mpim": false,
            "is_org_shared": false,
            "is_pending_ext_shared": false,
            "is_private": false,
            "is_shared": false,
            "last_read": "1588078223.000500",
            "name": "<redacted>",
            "name_normalized": "<redacted>",
            "num_members": 3,
            "parent_conversation": null,
            "pending_connected_team_ids": [],
            "pending_shared": [],
            "previous_names": [],
            "purpose": {
                "creator": "",
                "last_set": 0,
                "value": ""
            },
            "shared_team_ids": [
                "<redacted>"
            ],
            "topic": {
                "creator": "",
                "last_set": 0,
                "value": ""
            },
            "unlinked": 0
        },
        "client_msg_id": "<redacted>",
        "display_as_bot": false,
        "files": [
            {
                "created": 1588128385,
                "display_as_bot": false,
                "edit_link": "<redacted>",
                "editable": true,
                "external_type": "",
                "filetype": "text",
                "has_rich_preview": false,
                "id": "<redacted>",
                "is_external": false,
                "is_public": true,
                "is_starred": false,
                "lines": 2,
                "lines_more": 0,
                "mimetype": "text/plain",
                "mode": "snippet",
                "name": "test file",
                "permalink": "<redacted>",
                "permalink_public": "<redacted>",
                "pretty_type": "Plain text",
                "preview": "<redacted>",
                "preview_highlight": "<div class=\"CodeMirror cm-s-default CodeMirrorServer\" oncopy=\"if(event.clipboardData){event.clipboardData.setData('text/plain',window.getSelection().toString().replace(/\\u200b/g,''));event.preventDefault();event.stopPropagation();}\">\n<div class=\"CodeMirror-code\">\n<div><pre><redacted></pre></div>\n<div><pre></pre></div>\n</div>\n</div>\n",
                "preview_is_truncated": false,
                "public_url_shared": false,
                "size": 22,
                "timestamp": 1588128385,
                "title": "test file",
                "url_private": "<redacted>",
                "url_private_download": "<redacted>",
                "user": "<redacted>",
                "username": ""
            }
        ],
        "text": "Uploading test file",
        "ts": "1588128387.000700",
        "type": "message",
        "upload": true,
        "user": "<redacted>",
        "user_data": {
            "color": "e7392d",
            "deleted": false,
            "has_2fa": false,
            "id": "<redacted>",
            "is_admin": false,
            "is_app_user": false,
            "is_bot": false,
            "is_owner": false,
            "is_primary_owner": false,
            "is_restricted": false,
            "is_ultra_restricted": false,
            "name": "<redacted>",
            "profile": {
                "avatar_hash": "<redacted>",
                "display_name": "Animesh",
                "display_name_normalized": "Animesh",
                "image_192": "<redacted>",
                "image_24": "<redacted>",
                "image_32": "<redacted>",
                "image_48": "<redacted>",
                "image_512": "<redacted>",
                "image_72": "<redacted>",
                "phone": "",
                "real_name": "Animesh",
                "real_name_normalized": "Animesh",
                "skype": "",
                "status_emoji": "",
                "status_expiration": 0,
                "status_text": "",
                "status_text_canonical": "",
                "team": "<redacted>",
                "title": ""
            },
            "real_name": "Animesh",
            "team_id": "<redacted>",
            "tz": "Asia/Kolkata",
            "tz_label": "India Standard Time",
            "tz_offset": 19800,
            "updated": 1588082878
        }
    },
    "origin": "https://slack.com/<redacted>",
    "perceval_version": "0.13.0",
    "search_fields": {
        "channel_id": "<redacted>",
        "channel_name": "testapi",
        "item_id": "<redacted>"
    },
    "tag": "https://slack.com/<redacted>",
    "timestamp": 1588135884.542462,
    "updated_on": 1588128387.0007,
    "uuid": "d89baf53b5466210a4dea55f60c32f6f92fec434"
}
```

I think the files should be enriched as a sperate document similar to the comments on Github. This will make it easier for someone to create visualizations specific to file data(data table with all files and its URL, etc).

### PR DESCRIPTION
**Pull request by animeshk08**
Fixes: #867 
This commit removes enriching single files,
since a message can contain multiple files.
Instead, two new metrics, number of files and
file size per message have been included.

The schema has been updated.
The tests have been updated accordingly.

Signed-off-by: Animesh Kumar <animuz111@gmail.com>

### CODE REVIEW

File: grimoire_elk/enriched/slack.py [hunk 1]
```diff
@@ -139,16 +139,12 @@ def get_rich_item(self, item):
...
-            eitem['file_mode'] = message['file']['mode']
-            eitem['file_is_public'] = message['file']['is_public']
-            eitem['file_is_external'] = message['file']['is_external']
-            eitem['file_id'] = message['file']['id']
-            eitem['file_is_editable'] = message['file']['editable']
+        if 'files' in message:
+            eitem['number_files'] = len(message['files'])
+            message_file_size = 0
+            for file in message['files']:
+                message_file_size += file['size']
```
Comment (valeriocos) [inline grimoire_elk/enriched/slack.py:146]: file['size'] could be change to file.get('size', 0). This would avoid possible key errors. WDYT?

**Review by valeriocos — COMMENTED** [review]
Hi @animeshk08 , thank you for this PR!

The code looks OK (I left a comment). If possible, could you update the slack schema to the latest format (e.g., https://github.com/chaoss/grimoirelab-elk/blob/master/schema/git.csv)?

Thanks!

**Comment by animeshk08** [comment]
Hi @valeriocos, I have made the updates :)

**Comment by valeriocos** [comment]
many thanks @animeshk08 !!

**Review by valeriocos — APPROVED** [review]
LGTM, thanks @animeshk08

**Pull request merged**